tox.ini

```diff
@@ -3,9 +3,15 @@ skipsdist = True
 usedevelop = True
 
 envlist = 
-    py{37}-dj{32}-wt{215,216}
-    py{38,39,310}-dj{32,40}-wt{216,30,40}
-    py{39,310}-dj{41}-wt{40,41}
+    py{38,39,310}-dj{32,41,42}-wt{41,42,50}
+    py{311}-dj{41,42}-wt{41,42,50}
```

Comment:
If this can be condensed, this is good, but for some reason, it fails for me in another package:
https://github.com/katdom13/wagtailmenus/actions/runs/5217610335/jobs/9417615529

it seems like Wagtail 4.1 and 4.2 don't really support Django 4.2

Reply:
@katdom13 You are right, I made a change thanks.